app/src/main/res/values-it/strings.xml

```diff
@@ -21,148 +21,148 @@
-    <string name="downloading_csv">"CSV wird heruntergeladen"</string>
+    <string name="downloading_csv">"Scarica il file CSV"</string>
     <string name="handshakes_heading_homescreen">" "</string>
-    <string name="handshakes_title_homescreen">"Begegnungen"</string>
+    <string name="handshakes_title_homescreen">"Incontri"</string>
     <string name="inform_heading_homescreen">" "</string>
-    <string name="inform_title_homescreen">"Verdacht auf eine Corona Erkrankung"</string>
-    <string name="inform_text_homescreen">"Informiere Personen, denen du begegnet bist, wenn du Symptome verspürst oder positiv getestet wurdest."</string>
-    <string name="inform_button_title">"Erkrankung melden"</string>
-    <string name="inform_button_title_again">"Erneut informieren"</string>
-    <string name="inform_last_meldung_text">"Letzte erfolgreiche Meldung am {date}"</string>
+    <string name="inform_title_homescreen">"Sospetto COVID-19"</string>
+    <string name="inform_text_homescreen">"Informa le persone che hai recentemente incontrato che hai i sintomi o il test al COVID-19 è risultato positivo."</string>
+    <string name="inform_button_title">"Segnala adesso"</string>
+    <string name="inform_button_title_again">"Aggiorna la segnalazione"</string>
+    <string name="inform_last_meldung_text">"Ultima segnalazione effettuata {date}"</string>
 
     <!-- Erstes Tab -->
-    <string name="tab_theapp_title">"Die App"</string>
+    <string name="tab_theapp_title">"L'App"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_aktuell_title">"Aktuell"</string>
+    <string name="tab_aktuell_title">"Adesso"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_verhalten_title">"Verhalten"</string>
+    <string name="tab_verhalten_title">"Comportamento"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_prinzip_title">"Prinzip"</string>
+    <string name="tab_prinzip_title">"Principio"</string>
 
     <!-- Abbrechen -->
-    <string name="cancel">"Abbrechen"</string>
+    <string name="cancel">"Annulla"</string>
 
     <!-- Schliessen -->
-    <string name="close">"Schliessen"</string>
+    <string name="close">"Chiudi"</string>
 
     <!-- App Name ohne COVID-19 -->
     <string name="app_name_star">"Next Step"</string>
 
     <!-- Info Text ohne COVID-19 -->
-    <string name="homescreen_handshakes_module_info_label_star">"Zur Zeit hat kein Nutzer, der kürzlich in deiner Nähe war, ein Signal aktiviert."</string>
+    <string name="homescreen_handshakes_module_info_label_star">"Non ci sono segnalazioni di contatti con persone infette."</string>
 
     <!-- Text bei aktivierung ohne COVID-19 -->
-    <string name="homescreen_handshakes_module_gemeldet_label_star">"Ein Nutzer, der kürzlich in deiner Nähe war, hat ein Signal aktiviert."</string>
+    <string name="homescreen_handshakes_module_gemeldet_label_star">"Una persona con cui sei stato recentemente in conttato ha inviato una segnalazione."</string>
```

Comment:
Una persona con cui sei stato in contatto ha inviato una segnalazione

Reply:
(Version with "recently"): Una persona con cui sei stato in contatto recentemente ha inviato una segnalazione